mon-pix/app/routes/assessments/checkpoint.js

```diff
@@ -4,14 +4,19 @@ import RSVP from 'rsvp';
 export default Route.extend({
 
   afterModel(assessment) {
-
     return RSVP.hash({
       campaigns: this.get('store').query('campaign', { filter: { code: assessment.codeCampaign } }),
-      smartPlacementProgression: assessment.belongsTo('smartPlacementProgression').reload()
+      smartPlacementProgression: assessment.belongsTo('smartPlacementProgression').reload(),
+      answers: assessment.hasMany('answers').reload(),
     }).then((hash) => {
       assessment.set('campaign', hash.campaigns.get('firstObject'));
       return assessment;
     });
   },
 
+  setupController(controller, model) {
```

Comment:
Quelle est la raison de ce setupController ? Le controller a déjà accès au model et donc answersSinceLastCheckpoints.
Sinon vu notre implémentation je dirais que totalPixForFiveNewAnswer devrait être calculé dans une computedProperty dans le model assessment.

Reply:
On était partie pour utiliser le model, mais faire un this.get('model').get('answersSinceLastCheckpoints') ca nous semblait lourd et on trouvait que c'était + compréhensible coté controller avec un nom de variable.
A voir effectivement si on peut le remettre dans les computedProperty de l'assessment, en gérant bien la mise à jour des info